File: java/iniciante/URI1021.java
package URI;

import java.util.Scanner;

public class URI1021 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double value = scanner.nextDouble();
        double cont100, cont50, cont20, cont10, cont5, cont2, moeda100, moeda50, moeda25, moeda10, moeda5, moeda1;

        cont100 = value / 100;
        value %= 100;

        cont50 = value / 50;
        value %= 50;

        cont20 = value / 20;
        value %= 20;

        cont10 = value / 10;
        value %= 10;

        cont5 = value / 5;
        value %= 5;

        cont2 = value / 2;
        value %= 2;

        moeda100 = value;
        value %= 1;

        value = value * 100;

        moeda50 = value / 50;
        value %= 50;

        moeda25 = value / 25;
        value %= 25;

        moeda10 = value / 10;
        value %= 10;

        moeda5 = value / 5;
        value %= 5;

        moeda1 = value;

        System.out.println("NOTAS:");
        System.out.printf("%.0f nota(s) de R$ 100.00\n", Math.floor(cont100));
        System.out.printf("%.0f nota(s) de R$ 50.00\n", Math.floor(cont50));
        System.out.printf("%.0f nota(s) de R$ 20.00\n", Math.floor(cont20));
        System.out.printf("%.0f nota(s) de R$ 10.00\n", Math.floor(cont10));
        System.out.printf("%.0f nota(s) de R$ 5.00\n", Math.floor(cont5));
        System.out.printf("%.0f nota(s) de R$ 2.00\n", Math.floor(cont2));
        System.out.println("MOEDAS:");
        System.out.printf("%.0f moeda(s) de R$ 1.00\n", Math.floor(moeda100));
        System.out.printf("%.0f moeda(s) de R$ 0.50\n", Math.floor(moeda50));
        System.out.printf("%.0f moeda(s) de R$ 0.25\n", Math.floor(moeda25));
        System.out.printf("%.0f moeda(s) de R$ 0.10\n", Math.floor(moeda10));
        System.out.printf("%.0f moeda(s) de R$ 0.05\n", Math.floor(moeda5));
        System.out.printf("%.0f moeda(s) de R$ 0.01\n", Math.floor(moeda1));

    }
}

File: python/Iniciante/URI1021.py
nota_100, nota_50, nota_20, nota_10, nota_5, nota_2 = 0, 0, 0, 0, 0, 0
moeda_100, moeda_50, moeda_25, moeda_10, moeda_5, moeda_1 = 0, 0, 0, 0, 0, 0

n = float(input())

centavos_de_n = (n - int(n)) * 100

n_inicial = n

nota_100 = n // 100
n = n % 100

nota_50 = n // 50
n = n % 50

nota_20 = n // 20
n = n % 20

nota_10 = n // 10
n = n % 10

nota_5 = n // 5
n = n % 5

nota_2 = n // 2
n = n % 2

moeda_100 = int(n)

moeda_50 = centavos_de_n // 50
centavos_de_n = centavos_de_n % 50

moeda_25 = centavos_de_n // 25
centavos_de_n = centavos_de_n % 25

moeda_10 = centavos_de_n // 10
centavos_de_n = centavos_de_n % 10

moeda_5 = centavos_de_n // 5
centavos_de_n = centavos_de_n % 5

moeda_1 = centavos_de_n // 1
centavos_de_n = centavos_de_n % 1

print(f"NOTAS:\n{nota_100:.0f} nota(s) de R$ 100.00\n{nota_50:.0f} nota(s) de R$ 50.00\n{nota_20:.0f} nota(s) de R$ 20.00\n{nota_10:.0f} nota(s) de R$ 10.00\n{nota_5:.0f} nota(s) de R$ 5.00\n{nota_2:.0f} nota(s) de R$ 2.00")

print(f"MOEDAS:\n{moeda_100:.0f} moeda(s) de R$ 1.00\n{moeda_50:.0f} moeda(s) de R$ 0.50\n{moeda_25:.0f} moeda(s) de R$ 0.25\n{moeda_10:.0f} moeda(s) de R$ 0.10\n{moeda_5:.0f} moeda(s) de R$ 0.05\n{moeda_1:.0f} moeda(s) de R$ 0.01")